deploy.py

```diff
@@ -128,5 +156,25 @@ def create_control_node(config):
     # Upload all files in src/ to the bucket
     upload_cluster_scripts(config)
 
-    # Finally, deploy the control node cloudformation
-    create_control_node(config)
\ No newline at end of file
+    # Finally, deploy the control node CloudFormation
+    create_control_node(config)
+
+    name = stack_name(config)
+    print("Waiting for your CloudFormation to finish contructing")
+    cloudformation.get_waiter('stack_create_complete').wait(StackName=name)
```

Comment:
Super!! I love to see a waiter here... the code feels definitely cleaner than before...